src/ploigos_step_runner/step_implementer.py

```diff
@@ -299,6 +299,10 @@ def _required_config_or_result_keys():
             that are required before running the step.
         """
 
+    @staticmethod
+    def _str2bool(var):
```

Comment:
@redhat-snelson can we actually remove this and just use https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool where needed?

Reply:
Yeah, I have no problem with that. I originally attempted to avoid adding additional dependencies, _but_ I think we can expect a normal/typical Python 3.x install to have it.